submanifests/optional.yaml

```diff
@@ -34,7 +34,7 @@ manifest:
       groups:
         - optional
     - name: sof
-      revision: e7cb489d430dc2181e4a5f7f953ed1eaeec6668d
+      revision: pull/35/head
```

Comment:
This isn't the right way to do this.  The fix needs to merge first, otherwise these YAML files become constant games of ping-pong chasing upstream patches.

Reply:
This is just to have the fix tested with Zephyr's CI before it's merged in the external module.

Reply:
The SOF macro `IS_ALIGN()` is used in only 4 places! Just rename it to `SOF_ALIGN()` as requested by https://docs.zephyrproject.org/latest/contribute/coding_guidelines/index.html#rule-a-3-macro-name-collisions and make your forward, backward and backflip manifest life much easier.